sdk/parents/azure-data-sdk-parent/pom.xml

```diff
@@ -325,6 +325,18 @@
             <failOnError>false</failOnError>
           </configuration>
         </plugin>
+        <plugin>
```

Comment:
@vcolin7 - So you're aware of why this particular change, in the second commit, was necessary. When this was changed back in April it was moved under the reporting instead of the pluginManagement. The reason this was passing is because when it couldn't find the data parent, it pulled the version from Maven and because that parent path was wrong it pulled the azure-sdk-parent from maven which had the correct entry.

Reply:
Thanks for the explanation Jim! I was wondering why the my IDE would show the path all in red.